lib/util.js

```diff
@@ -194,6 +194,7 @@ function inspect(obj, opts) {
   if (ctx.colors) ctx.stylize = stylizeWithColor;
   if (ctx.maxArrayLength === undefined) ctx.maxArrayLength = kDefaultMaxLength;
   if (ctx.maxArrayLength === null) ctx.maxArrayLength = Infinity;
+  if (ctx.breakLength === undefined) ctx.breakLength = 60;
```

Comment:
`Infinity` is documented to make the object print on a single line with no breaks. `null`, or any value for that matter, will be plugged in for `breakLength` in the `length > breakLength` comparison.